src/libraries/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/ContentEncryptionAlgorithmTests.cs

```diff
@@ -12,7 +12,7 @@ namespace System.Security.Cryptography.Pkcs.EnvelopedCmsTests.Tests
     public static partial class ContentEncryptionAlgorithmTests
     {
         public static bool SupportsRc2 => PlatformSupport.IsRC2Supported;
-        public static bool SupportsRc4 => PlatformDetection.IsWindows;
+        public static bool SupportsRc4 => TargetFrameworkHelpers.TargetsWindows;
```

Comment:
I don't think we should take this change, at all.

A change which makes the VS runner see the same assets as the CLI runner, sure.  But random extra hacking on the side, no.  It will just leave behind a continual specter of "should this use TargetsWindows, or IsWindows?".